grr/server/grr_response_server/gui/ui/components/client_add_label_dialog/client_add_label_dialog.ng.html

```diff
@@ -0,0 +1,24 @@
+<div mat-dialog-title class="title">Adding a label</div>
+<div mat-dialog-content class="content">
+  <p>You can add a new label or select from existing labels.</p>
+  <mat-form-field class="label-field" appearance="outline">
+    <mat-label>Start typing label name</mat-label>
+    <input matInput
+      cdkFocusInitial
+      (keyup.enter)="onAddClick()"
+      [formControl]="newLabelInputControl"
+      [matAutocomplete]="auto">
+
+    <mat-autocomplete #auto="matAutocomplete">
+      <mat-option *ngFor="let option of filteredCurrentLabels | async" [value]="option">
+        {{option}}
+      </mat-option>
+    </mat-autocomplete>
+
+    <mat-error *ngIf="newLabelInputControl.invalid">This label is already present!</mat-error>
```

Comment:
Use `hasError('alreadyPresentLabel')` instead of `invalid` to only show this error message for the validator that should trigger it.

Reply:
Good idea!